utils/genesis/src/lib.rs

```diff
@@ -145,7 +145,7 @@ where
     // Update head with snapshot header tipset
     sm.chain_store().set_heaviest_tipset(ts.clone())?;
 
-    sm.blockstore().flush()?;
+    //sm.blockstore().flush()?;
```

Comment:
We still need this flush, unfortunately.

Reply:
Will add back.